catroid/src/main/java/org/catrobat/catroid/content/bricks/SetLookBrick.java

```diff
@@ -132,7 +136,7 @@ public void onNothingSelected(AdapterView<?> arg0) {
 
 		if (getSprite().getName().equals(context.getString(R.string.background))) {
 			TextView textField = (TextView) view.findViewById(R.id.brick_set_look_prototype_text_view);
-			textField.setText(R.string.brick_set_background);
+			textField.setText(getTextForView());
```

Comment:
why did this move to a protected method which is only used here and below?

Reply:
The method is overridden in the LegoEv3ShowImageBrick. Therefore I don't have to override the whole getView and getPrototypeView in LegoEv3ShowImageBrick but only the getTextForView method.